packages/components/src/checkbox-control/test/index.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import BaseCheckboxControl from '..';
+import type { CheckboxControlProps } from '../types';
+
+const getInput = () => screen.getByTestId( 'checkbox' );
+
+const CheckboxControl = ( props: Omit< CheckboxControlProps, 'onChange' > ) => {
+	return (
+		<BaseCheckboxControl
+			onChange={ noop }
+			{ ...props }
+			data-testid="checkbox"
```

Comment:
This line is not necessary anymore, after applying the suggestion from the previous comment

```suggestion
```